### ISSUE TITLE
Global `keep_docs_dir` flag ignored?

### ISSUE THREAD

**Issue by Sebastian-0** [jdoiro3/mkdocs-multirepo-plugin#87]
I had an issue where I set `keep_docs_dir` globally but the docs dir was still removed when I import other repos.

Looking in your code `structure.py:301` we can see:
```python
def keep_docs_dir(self, global_config_val: bool = False):
    return self._keep_docs_dir and (self._keep_docs_dir or global_config_val)
```
The global value is ignored here since you `and` with `self._keep_docs_dir`. Was it intended to be like below?
```python
def keep_docs_dir(self, global_config_val: bool = False):
    return self._keep_docs_dir or global_config_val
```
I can make a PR for this if you want.

**Comment by Sebastian-0** [comment]
I wrote a PR for this, see: https://github.com/jdoiro3/mkdocs-multirepo-plugin/pull/89 

I went with option 1 but extended it a bit, see the PR. I also added docs to the doc string like you suggested.

**Comment by jdoiro3** [comment]
https://github.com/jdoiro3/mkdocs-multirepo-plugin/pull/89 is now merged at part of `v0.6.1`.
